### PR TITLE
[chore] add autocli for oracle module

### PR DESCRIPTION
**Pull request by taobun**
## Implementation details
- Generate pulsar proto for autocli
- Change AllocationToken to return error on beginblock instead of panic

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [ ] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

### CODE REVIEW

**Review by RogerKSI — APPROVED** [review]

File: x/oracle/keeper/validator_status_test.go [hunk 1]
```diff
@@ -147,7 +147,8 @@ func (suite *KeeperTestSuite) TestAllocateTokensAllActive() {
 			AllocateTokensToValidator(gomock.Any(), validators[0].Validator, (sdk.DecCoins)(nil)),
 	)
 
-	k.AllocateTokens(ctx, defaultVotes)
+	err = k.AllocateTokens(ctx, defaultVotes)
+	require.NoError(err)
```
Comment (colmazia) [inline x/oracle/keeper/validator_status_test.go:151]: use suite.Require().NoError(k.AllocateTokens(ctx, defaultVotes)) like above

File: x/oracle/autocli.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+					Short:     "Check report grant of reporter",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{
+						{ProtoField: "validator_address"},
+						{ProtoField: "reporter_address"},
+					},
+				},
+				{
+					RpcMethod:      "Reporters",
+					Use:            "reporters [validator-address]",
+					Short:          "Get an oracle request details",
```
Comment (colmazia) [inline x/oracle/autocli.go:68]: wrong short message

Comment (RogerKSI) [reply x/oracle/autocli.go]: fix

File: x/oracle/keeper/validator_status_test.go (hunk 1 above)
Comment (RogerKSI) [reply x/oracle/keeper/validator_status_test.go]: fixed

**Review by colmazia — APPROVED** [review]

**Pull request merged**